Apps/MAUI.Demo/AddPropertyContentPage.xaml

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Copyright?

Reply:
We don't have copyrights in xml files in our Forms app as well. Worth adding?